include/swift/AST/DiagnosticsSema.def

```diff
@@ -1973,8 +1973,8 @@ WARNING(protocol_usable_from_inline_warn,none,
         "protocol %select{refined|used}0 by '@usableFromInline' protocol "
         "should be '@usableForInline' or public", (bool))
 ERROR(protocol_property_must_be_computed_var,none,
```

Comment:
The next thing to do is split the dependent clause off into a note and attach the fixit there. This diagnostic is quite wordy now...